config/functions/faction_kill.py

```diff
@@ -14,6 +14,7 @@
 """
 
 config_file_path = '../environments/{}/database.json'
+#config_file_path = '../environments/{}/database.json'
```

Comment:
We can remove this line (and likely the line below it.

Reply:
Yep paths are a pain. We can pass them in on the command-line and in the model we can get the path from strapi to pass in